index.html

```diff
@@ -61,7 +61,7 @@
 
   <section id="getting-started">
     <aside>
-    <h3>Alright. How do I get started?</h3>
```

Comment:
"Alright" is a merged word, sure. That's not the point. The point is that by whatever historical accident, it is not accepted English usage as "altogether" and "already" are. The animosity toward "alright" is simply that it's a spelling error; the fact that it's a common error doesn't make it correct (see "miniscule" for a similar case). As an error, it doesn't belong in any proper English writing, formal or not.